bin/commands/runs.js

```diff
@@ -112,6 +112,7 @@ module.exports = function run(args, rawArgs) {
       // warn if specFiles cross our limit
       utils.warnSpecLimit(bsConfig, args, specFiles, rawArgs);
       markBlockEnd('preArchiveSteps');
+      markBlockStart('zip');
       markBlockStart('checkAlreadyUploaded');
```

Comment:
let's mark this `zip.checkAlreadyUploaded` and similarly for other blocks below this